eng/targets/ResolveReferences.targets

```diff
@@ -85,7 +85,7 @@
       <_ReferenceTemp Remove="@(_ReferenceTemp)" />
 
       <!-- Identify if any references were present in the last release of this package, but have been removed. -->
-      <UnusedBaselinePackageReference Include="@(BaselinePackageReference)" Exclude="@(Reference);@(_ProjectReferenceByAssemblyName)" />
+      <UnusedBaselinePackageReference Include="@(BaselinePackageReference)" Exclude="@(Reference);@(_ProjectReferenceByAssemblyName);@(PackageReference)" />
```

Comment:
This is okay. Razor projects have to use a PackageReference to Microsoft.AspNetCore.Razor.Design because ProjectReferences are not a suitable substitute for this particular package. We want to preserve the dependencies of generated nuspec's.